.github/workflows/verify_samples.yml

```diff
@@ -0,0 +1,51 @@
+name: Build Samples
+
+on: 
+  push:
+    branches: master
```

Comment:
Recommend to change this to `'*'`, otherwise forked repos will only build master as well.

Reply:
The reason I set it like this is to avoid having it run twice if someone makes a self-PR from within the Project. I can change it still if you wish.

Reply:
I currently don't have an opinion on this.

> if someone makes a self-PR from within the Project.

I strongly doubt we'd do that.
It also has the negative side-effect of forks having the temporary branch, which is something we avoid.

---

I have yet to try GitHub Actions.

- Does GitHub Actions allow deleting of builds? (if not, we might want to keep the "master" variant to avoid accidental pushes)
- Does it allow manual configuration overrides outside of this yml? (if so, we could fix this upstream)

Reply:
> Does GitHub Actions allow deleting of builds?

From what I can tell, no.

Reply:
> Does it allow manual configuration overrides outside of this yml?

If you're talking about a single run where the configuration is changed, no.

Reply:
I'm not sure if we are talking about the same thing. So I'll explain what I was thinking:

I believe Travis and AppVeyor allow to specify a configuration in their Web-UI. This allows projects to ignore hostile changes in PRs (which might affect the visibility of the build) or to allow storing a CI configuration for forks, while using a different configuration upstream.

So if GitHub Actions allows overriding configuration fields in a web UI, then we can keep our CI config in the git repository configured for users of forks (build `*`); our own repository would be using a Web-UI override to only  build `master` (although that is pointless if that override would also be included in forks).

Reply:
Ah then no, it seems it does not allow such things.